lib/std/Build.zig

```diff
@@ -1968,9 +2083,11 @@ pub fn getInstallPath(b: *Build, dir: InstallDir, dest_rel_path: []const u8) []c
     return b.pathResolve(&.{ base_dir, dest_rel_path });
 }
 
+/// Represents a package dependency defined in `build.zig.zon`
```

Comment:
Redundant and not entirely accurate; a Dependency need not always be represented in the build.zig.zon file, as it can be derived via `dependencyFromBuildZig`.